From 67131a2be333c2f632d97b00ad6afb8508a50e79 Mon Sep 17 00:00:00 2001
From: Adarsh0127-Elite <adarshmoharana027@gmail.com>
Date: Wed, 30 Sep 2026 15:37:16 +0000
Subject: [PATCH 8/9] MiuiCamera: Unlock Eye tracking focus

Force capability check for eye tracking focus to return true.

Signed-off-by: Adarsh0127-Elite <adarshmoharana027@gmail.com>
---
 smali_classes4/n9/g.smali | 40 +++------------------------------------
 1 file changed, 3 insertions(+), 37 deletions(-)

diff --git a/smali_classes4/n9/g.smali b/smali_classes4/n9/g.smali
index 7a23413c..a2602630 100644
--- a/smali_classes4/n9/g.smali
+++ b/smali_classes4/n9/g.smali
@@ -13191,45 +13191,11 @@
 .end method
 
 .method public static d1(Ln9/f;)Z
-    .locals 2
-
-    if-eqz p0, :cond_1
-
-    iget-object v0, p0, Ln9/f;->b0:Ljava/lang/Boolean;
-
-    const/4 v1, 0x1
-
-    if-nez v0, :cond_0
-
-    invoke-virtual {p0}, Ln9/f;->f0()[I
-
-    move-result-object v0
-
-    invoke-static {v1, v0}, LWr/e;->j(I[I)Z
-
-    move-result v0
-
-    invoke-static {v0}, Ljava/lang/Boolean;->valueOf(Z)Ljava/lang/Boolean;
-
-    move-result-object v0
-
-    iput-object v0, p0, Ln9/f;->b0:Ljava/lang/Boolean;
-
-    :cond_0
-    iget-object p0, p0, Ln9/f;->b0:Ljava/lang/Boolean;
-
-    invoke-virtual {p0}, Ljava/lang/Boolean;->booleanValue()Z
-
-    move-result p0
-
-    if-eqz p0, :cond_1
-
-    return v1
+    .locals 1
 
-    :cond_1
-    const/4 p0, 0x0
+    const/4 v0, 0x1
 
-    return p0
+    return v0
 .end method
 
 .method public static d2(Ln9/f;)Z
-- 
2.43.0

